resources/views/ItemList/Components/Filter/ItemFilterList.twig

```diff
@@ -1,75 +0,0 @@
-                                v-if="facet.id === 'cat' ? showCategoryFilter && facet.isMain : facet.isMain"
-                                :current-category="currentCategory"
-                                :show-category-filter="showCategoryFilter"
-                                :facet="facet"
-                                :key="facet.id"
-                            ></findologic-item-filter>
-                        </div>
-                    </client-only>
-                </div>
-                <div class="row">
-                    <a class="btn btn-link filter-toggle" data-toggle="collapse" href="#filterCollapse"
-                       aria-controls="filterCollapse">
-                        <i class="fa fa-sliders default-float"
-                           aria-hidden="true"></i> {{ trans("Findologic::Template.itemFilter") }}
-                    </a>
-                </div>
-            </div>
-        </div>
-        <div v-else class="ml-0">
-            <div class="container-max component-loading" :class="{ 'isLoading': isLoading }">
-                <div class="row">
-                    <a class="btn btn-link filter-toggle no-main-filters-filter-toggle" data-toggle="collapse"
-                       href="#filterCollapse" aria-expanded="false" aria-controls="filterCollapse">
-                        <i class="fa fa-sliders default-float"
-                           aria-hidden="true"></i> {{ trans("Findologic::Template.noMainFiltersItemFilter") }}
-                    </a>
-                </div>
-            </div>
-        </div>
-        <div class="ml-0 filter-collapse collapse" id="filterCollapse">
-            <div class="container-max component-loading page-content mb-5" :class="{ 'isLoading': isLoading }">
-                <div class="card-columns row">
-                    <client-only>
-                        <div class="w-100">
-                            <findologic-item-filter
-                                v-bind:filtersPerRow="{{ config("Findologic.filters_per_row") }}"
-                                v-bind:fallbackImageColorFilter="'{{ plugin_path("Findologic") }}/images/no-picture.png'"
-                                v-bind:fallbackImageImageFilter="'{{ plugin_path("Findologic") }}/images/no-picture-2.png'"
-                                v-bind:show-selected-filters-count="{{ config('Findologic.show_selected_filter_count') }}"
-                                v-for="facet in facets"
-                                v-if="!facet.isMain"
-                                :current-category="currentCategory"
-                                :show-category-filter="showCategoryFilter"
-                                :facet="facet"
-                                :key="facet.id"
-                            ></findologic-item-filter>
-                        </div>
-                    </client-only>
-                </div>
-            </div>
-        </div>
-    </div>
```

Comment:
The filters are currently not split up into main/other filters. We will need to copy this part to the FindologicFilterWrapper.vue and make sure the other filters are expandable.

Have a look at the ItemFilterList.vue component.